### ISSUE TITLE
Maps APP

### ISSUE THREAD

**Issue by tuliogit** [nylira/prism-break#382]
Before starting: you forgot to mention Yahoo Maps on the left collumn, grouped with Google Maps, Bing and Apple Maps.
Now, about the alternative: OSM is an alternative to the BASEMAP. It isn't an alternative, for example, to Google Android Maps App. OSM website mentions a long and exaustive list of apps that use OSM basemap. But many of them, despite being free and open, are soft of unfinished, just like the software I suggested in issue #298 .
I actually wouldn't know an alternative to Maps App, since I tried several ones and didn't like any.
If anybody knows a small list of alternatives, it would be nice to have it in the website.
By the way, for those who never thought about it: it's been a long time I uninstalled Google Maps App from my phone, because I realized that, even deactivating "Latitude", it still send some data (I'm sure it was location data) to Google servers, even if I didn't use the app at all. This could be in the "Comment" session.
Thanks.
